### DISCUSSION TITLE
REQUEST: Docker image & SVG logo

### DISCUSSION THREAD

**Discussion by wwwescape** [thomasnordquist/MQTT-Explorer#852] [General]
Please could you provide a Docker image so that we can self-host this inside a container? Also please could you provide an SVG logo that we could submit to https://github.com/walkxcode/dashboard-icons for use with https://github.com/gethomepage/homepage

**Comment by NetRat88** [comment]
A docker image would be great to have access through the web from all devices to this great tool!
Than it's also possible to integrate it to the Home-Assistant sidebar.

**Comment by Gandalf77bo** [comment]
Voici ce que j'ai trouvé:
https://hub.docker.com/r/smeagolworms4/mqtt-explorer

Importez l'image :
`docker pull smeagolworms4/mqtt-explorer`

Puis le stack :
`version: '3'
services:
 mqtt-explorer:
      container_name: mqtt-explorer
      image: smeagolworms4/mqtt-explorer
      hostname: mqtt-explorer
      ports:
        - 9002:9002
      environment:
        - HTTP_PORT=9002
        - CONFIG_PATH=/mqtt-explorer/config
        - TZ=Europe/Paris
      volumes:
        - ./config:/mqtt-explorer/config
        - /etc/timezone:/etc/timezone:ro`

**Reply by jadero** [reply]
This build is good but not perfect. There are missing icons and the file selection menu (for tls certificates) does not open (must edit settings file instead).
